src/splitwisesdk/core/GroupMember.java

```diff
@@ -0,0 +1,15 @@
+package splitwisesdk.core;
+
+public class GroupMember {
+    private People member;
+    private double balance;
+    private String groupId;
+
```

Comment:
add constructor
GroupMember(People people, String groupId) {
   this.member = people;
   this.groupId = groupId;
}